tailwind.config.js

```diff
@@ -1,122 +1,120 @@
-import defaultTheme from 'tailwindcss/defaultTheme';
```

Comment:
Why is that necessary? Just auto format?

Reply:
yes typo. Based on .editorconfig. Default is double-quotes.